src/components/OrderBookWidget.tsx

```diff
@@ -143,13 +171,14 @@ const draw = (
     const processed = processData(data.bids, baseToken, quoteToken, Offer.Bid).concat(
       processData(data.asks, baseToken, quoteToken, Offer.Ask),
     )
-    processed.sort((lhs, rhs) => lhs.price - rhs.price)
+    processed
+      // cast as number to sort...
+      .sort((lhs, rhs) => +lhs.price - +rhs.price)
+      // show as string
+      .map(processedItem => ({ ...processedItem, price: checkPriceAndConvert(processedItem.price as number) }))
```

Comment:
can `price` even be string? I looked at `processData` and it seems price is always number. Or is there a reason?

Reply:
I'm afraid of casting to number in line 178